out_newrelic.go

```diff
@@ -48,6 +48,10 @@ func FLBPluginFlushCtx(ctx, data unsafe.Pointer, length C.int, tag *C.char) int
 	// Create Fluent Bit decoder
 	dec := output.NewDecoder(data, int(length))
 
+	// Get New Relic Client
+	id := output.FLBPluginGetContext(ctx).(string)
+	nrClient := nrClientRepo[id]
```

Comment:
If we opt to having a separate `DataFormatConfig` config struct, then we'll need to keep a `dataFormatConfigRepo` as well.

Reply:
...or just store the whole `PluginConfig`, even though this would represent some unnecessary memory footprint. Maybe let's just have the additional `dataFormatConfigRepo`, for now.